src/test/java/uk/nhs/ctp/service/search/SearchParametersTransformerTest.java

```diff
@@ -77,8 +77,10 @@ public void createSearchParametersFromSettings() {
     assertThat(searchParameters.getQuery(), is("triage"));
     assertThat(searchParameters.getJurisdiction(), is("GB"));
     assertThat(searchParameters.getContextValueCode(),
-        containsInAnyOrder("user$http://hl7.org/fhir/valueset-provider-taxonomy.html|Patient",
-            "setting$http://hl7.org/fhir/valueset-provider-taxonomy.html|online", "task$cdss/supplier/stub|example"));
+        containsInAnyOrder(
+            "user$http://hl7.org/fhir/valueset-provider-taxonomy.html|Patient",
+            "725221000000100$https://fhir.nhs.uk/STU3/CodeSystem/UEC-CommunicationChannel-1|online",
```

Comment:
This does seem a bit weird now I see the snomed code there. I'll confirm with Afzal around the setting code system, but I think this is a reasonable compromise since we don't have another candidate code system to use.